### PR TITLE
Added proposed changes

### PR DESCRIPTION
**Pull request by nikcbg**
I have added the proposed changes and removed the note I initially created. 

Please feel free to add, change or edit anything that might make this more clear to understand.

## Labels

- [ ] inaccuracy
- [x] clarification
- [ ] new docs
- [ ] cosmetic bug - fixing broken text or markup
- [ ] enhancement - changing the website's behavior/layout
- [ ] api - requires an update to the [changelog](https://www.terraform.io/docs/cloud/api/changelog.html)

### CODE REVIEW

**Review by laurapacilio — CHANGES_REQUESTED** [review]
Thank you so much! A couple nits for you here and then I think we'll be good to go!

**Comment by nikcbg** [comment]
Hi Laura, 

I reviewed the changes and everything looks good to me except for one thing. The force unlock cli command and the link you shared are for Terraform state file and not for Terraform workspace so it needs to be removed. Here is a link that is related to Terraform workspace lock/unlock: https://www.terraform.io/docs/cloud/workspaces/settings.html#locking. 

If you have any other questions please let me know.

**Comment by laurapacilio** [comment]
ahhh thank you! :-) I'm sorry about that! I updated my suggestion with the new link. Would you mind applying these changes now?

**Comment by nikcbg** [comment]
Hi @laurapacilio,

I've committed the changes you proposed and I think we are good to go.

Thanks.

File: content/source/docs/cloud/users-teams-organizations/permissions.html.md [hunk 1]
```diff
@@ -133,17 +133,11 @@ The "write" permission set is for people who do most of the day-to-day work of p
 
 - Apply runs
 - Lock and unlock workspace
+   -> **Note**: Typically, only the user who created the lock can unlock a workspace. Administrators and members of the "Owners" team can use [Force unlock] (/docs/cloud/workspaces/settings.html#locking) to override this an unlock a workspace.
```
Comment (laurapacilio) [inline content/source/docs/cloud/users-teams-organizations/permissions.html.md:136]:
```suggestion
- Lock and unlock workspace

    -> **Note**: Typically, only the user who created the lock can unlock a workspace. Administrators and members of the "Owners" team can use [Force unlock] (/docs/cloud/workspaces/settings.html#locking) to override this an unlock a workspace.
```

**Review by laurapacilio — APPROVED** [review]
Thank you so much for your patience getting this done!  I tested this locally, and I have one more suggested change for you to commit. This will ensure that the content displays properly on the site (we need certain spacing, etc. for bulleted lists like this).  I'm going to approve so I don't block you.  Could you just commit this last suggested change? Then, you can feel free to merge when you're ready 🚀

**Pull request merged**